src/NovelAudioService.cpp

```diff
@@ -0,0 +1,162 @@
+//
+// Created by krjoh on 29/08/2019.
+//
+
+#include "NovelAudioService.h"
+#include <iostream>
+#include <functional>
+
+namespace NovelRT {
+
+NovelAudioService::NovelAudioService(){
+  initializeAudio();
+  _nextChannel = 0;
+}
+
+bool NovelAudioService::initializeAudio(){
+  if (SDL_Init(SDL_INIT_AUDIO) < 0 )
+  {
+    std::cout << "ERROR: Cannot play audio!" << std::endl;
+    std::cout << "SDL Error: " << SDL_GetError() << std::endl;
+    return isInitialized;
```

Comment:
nit: You shouldn't use a local before it was declared nor before it was initialized. Just return the constant `false` or declare  it at the top and return once (which is more efficient codegen).

Reply:
I believe I corrected this - maybe a re-review of the new commit would confirm?